frontend/tailwind.config.js

```diff
@@ -8,19 +9,37 @@
 /** @type {import('tailwindcss').Config} */
 const plugin = require('tailwindcss/plugin');
 
+// eslint-disable-next-line no-undef
 module.exports = {
     content: ['./src/**/*.{js,jsx,ts,tsx}'],
     theme: {
         extend: {
             colors: {
-                react: {
-                    DEFAULT: '#282c34',
+                // Color thema of the application
+                // can be used in normal tailwind syntax, e.g.,
+                // bg-primary-dark for to use '#002884'
+                primary: {
+                    light: '#757ce8',
```

Comment:
In my opinion, these should be eg. `blue-500' and so on. Having 'primary-light' hides the actual color.

Reply:
But actually restricting the amount of available colors is good! TW ships with a quite extensive color set, dropping most of those is a good thing.